CHANGES

```diff
@@ -9,6 +9,10 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Added new method to gether entropy on VMS, based on SYS$GET_ENTROPY.
+     The presence of that system service is done in run-time.
```

Comment:
`s/that/this/`
`s/done in/performed at/`

Reply:
... and `s/gether/gather/`...